modules/tinymce/src/themes/silver/test/ts/browser/editor/bespoke/SilverBespokeButtonsTest.ts

```diff
@@ -157,11 +157,11 @@ describe('browser.tinymce.themes.silver.editor.bespoke.SilverBespokeButtonsTest'
       TinySelections.setCursor(editor, [ 0, 0 ], 'Fi'.length);
       assertEvent(1, 'Verdana');
       await MenuUtils.pOpenMenu('FontSelect', 'Verdana');
-      assertEvent(2, 'Verdana');
+      assertEvent(1, 'Verdana');
```

Comment:
Why did these need to change?

Reply:
Although... it looks like things are still a _little_ inconsistent, because we just had a test run fail due to another section of the file now only receiving one event in a similar scenario.